### DISCUSSION TITLE
(Question) Is there a way to add predicted annotations to compare to ground truth?

### DISCUSSION THREAD

**Discussion by Robotatron** [pylabel-project/pylabel#81] [Q&A]
Say I load my test set in COCO format with images and annotations.
Now I also would like to add predictions from several of my models. Those predictions are also in COCO format for model1 and model2.

Is there a way of doing this and visualizing the ground truth as well as predictions of two separate models?

**Comment by alexheat** [comment]
Hello @Robotatron , this is how I would do it. Refer to one of the sample notebook like this: https://github.com/pylabel-project/samples/blob/main/coco2voc.ipynb 

1. Import the groundtruth, model1, and model1 labels as seperate datasets 
2. Then visualize the same image in each of the datasets. For example:
```
display(dataset1.visualize.ShowBoundingBoxes("BloodImage_00315.jpg"))
display(dataset2.visualize.ShowBoundingBoxes("BloodImage_00315.jpg"))
display(dataset3.visualize.ShowBoundingBoxes("BloodImage_00315.jpg"))
```
